app/questionnaire/relationships_router.py

```diff
@@ -0,0 +1,38 @@
+from app.questionnaire.relationship_location import RelationshipLocation
+
+
+class RelationshipRouter:
+    def __init__(self, block_id, list_item_ids):
+        self.path = self._generate_relationships_routing_path(block_id, list_item_ids)
+
+    def can_access_location(self, location):
+        return location in self.path
+
+    def get_first_location(self):
+        return self.path[0]
```

Comment:
Getting when adding people and then answering No. Was working previously.
```
get_first_location
    return self.path[0]
IndexError: list index out of range
```

Reply:
This was possibly while the branch was broken. It's working now.